### ISSUE TITLE
ignore-branches-regex causes all branches to be ignored by default

### ISSUE THREAD

**Issue by billiams** [dc-ag/auto-assign-reviewers-from-team#103]
@NoNamePaul94: The new `ignore-granches-regex` argument default added in https://github.com/dc-ag/auto-assign-reviewers-from-team/commit/02b410e6e22cc19dd0fba9008bd86526d883fa6f causes all branches to be ignored if you don't set it or if you set it to be a blank string:
```
Run dc-ag/auto-assign-reviewers-from-team@v1.2.1
  with:
    repo-token: ***
    admin-repo-token: ***
    team: <redacted>
    amount: 1
    use-team-reviewer-settings: false
Branch matches ignore pattern, skipping!
```

We had to explicitly set the regex to match on empty strings to get it to work:

```yaml
ignore-branches-regex: '^(?![\s\S])'
```

**Comment by NoNamePaul94** [comment]
Fixed as of [v1.2.2](https://github.com/dc-ag/auto-assign-reviewers-from-team/releases/tag/v1.2.2)

**Comment by billiams** [comment]
@NoNamePaul94 thanks for the quick response!
